src/Microsoft.AspNetCore.WebHooks.Receivers.Salesforce/SalesforceWebHookAttribute.cs

```diff
@@ -17,12 +17,7 @@ namespace Microsoft.AspNetCore.WebHooks
     /// <see cref="System.Xml.Linq.XElement"/> or <see cref="SalesforceNotifications"/>.
     /// </para>
     /// <para>
-    /// The '<c>MS_WebHookReceiverSecret_SalesforceSoap</c>' configuration value contains Salesforce Organization IDs.
-    /// The Organizational IDs can be found at <see href="http://www.salesforce.com"/> under
-    /// <c>Setup | Company Profile | Company Information</c>.
-    /// </para>
```

Comment:
In another branch (not a PR yet), I correct this information and move it to the `MvcBuilder` / `MvcCoreBuilder` extension methods.